### PR TITLE
Clean package.json and .eslintrc

### PR DESCRIPTION
**Pull request by pdehaan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,45 +1,25 @@
...
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/mozilla/page-metadata-parser.git"
-  },
-  "keywords": [
-    "page",
-    "metadata",
-    "parser"
-  ],
```
Comment (pdehaan) [inline package.json]: Most of this noise was just me running [**fixpack**](http://npm.im/fixpack) which likes to A-z things.

File: package.json [hunk 2]
```diff
@@ -1,45 +1,25 @@
...
     "chai": "^3.5.0",
     "coveralls": "^2.11.9",
     "eslint": "^2.13.1",
-    "eslint-plugin-mozilla": "0.0.3",
+    "eslint-plugin-mozilla": "^0.0.3",
     "istanbul": "^0.4.4",
     "istanbul-instrumenter-loader": "^0.2.0",
     "jsdom": "^9.2.1",
     "karma": "^1.0.0",
+    "karma-chrome-launcher": "^1.0.1",
```
Comment (pdehaan) [inline package.json]: I noticed this was a dependency above, but **karma** and the other __karma-*__ modules were listed as devDependencies.

File: package.json [hunk 3]
```diff
@@ -49,5 +29,25 @@
...
+    "type": "git",
+    "url": "git+https://github.com/mozilla/page-metadata-parser.git"
+  },
+  "scripts": {
+    "cover": "cat ./coverage/lcov/lcov.info | coveralls",
+    "tdd": "npm run test:karma -- --no-single-run",
+    "test": "npm-run-all test:*",
+    "test:karma": "karma start",
+    "test:lint": "eslint .",
+    "test:mocha": "istanbul cover _mocha --report lcovonly -- tests/*.test.js -R spec"
```
Comment (pdehaan) [inline package.json]: Shortened to just `instanbul` instead of the more verbose `./node_modules/istanbul/lib/cli.js`.

**Comment by jaredkerim** [comment]
r+!

**Pull request merged**